### PR TITLE
Fix nwfilter_vm_start possitive_test new_filter ipset_test

### PR DESCRIPTION
**Pull request by chunfuwen**
Previous bug fix bring logic change for this case, need backout change for this
Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
previous change https://github.com/autotest/tp-libvirt/pull/2719 will bring logic change to this case, which is supposed not to be.
By strictly constrain code change to nwfilter_vm_start.possitive_test.new_filter.variable_notation only

**Comment by chunfuwen** [comment]
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.nwfilter_vm_start.possitive_test.new_filter.ipset_test

File: libvirt/tests/src/nwfilter/nwfilter_vm_start.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def run(test, params, env):
                     test.fail("Rum command '%s' failed" % check_cmd)
                 # This change anchors nwfilter_vm_start.possitive_test.new_filter.variable_notation case
                 # The matched destination could be ip address or hostname
-                if "iptables -L" in check_cmd and expect_match:
+                if "iptables -L" in check_cmd and expect_match and 'ACCEPT' in expect_match:
```
Comment (kylazhang) [inline libvirt/tests/src/nwfilter/nwfilter_vm_start.py:120]: Can it compatible for the old version?

Comment (chunfuwen) [reply libvirt/tests/src/nwfilter/nwfilter_vm_start.py:120]: Yes, this correct last fix error where this case will fall in this logic.

**Pull request merged**